### PR TITLE
Input is added twice in the OptiPngProcessor.

### PR DESCRIPTION
**Pull request by antoligy**
Thanks to @notrix for spotting this!

### CODE REVIEW

**Comment by antoligy** [comment]
Sorry to bother you again @lsmith77, would you mind looking at this one?

File: Imagine/Filter/PostProcessor/OptiPngPostProcessor.php [hunk 1]
```diff
@@ -74,7 +74,6 @@ public function process(BinaryInterface $binary)
         }
 
         $pb->add($input = tempnam(sys_get_temp_dir(), 'imagine_optipng'));
```
Comment (notrix) [inline Imagine/Filter/PostProcessor/OptiPngPostProcessor.php]: This line should be removed as $input is defined in line 62

Comment (antoligy) [inline Imagine/Filter/PostProcessor/OptiPngPostProcessor.php]: 👍

**Comment by lsmith77** [comment]
ah .. there it is already

**Comment by antoligy** [comment]
Yep, tests passing - Not sure how it slipped by in the rebase, I must have resolved one of the merges incorrectly the first.
I'm thinking it might (separately) be worth adding some tests to prevent this kind of regression.

**Pull request merged**